### PR TITLE
Added docker file changes for the cloudfeeds -atomhopper

### PR DESCRIPTION
**Pull request by shub6691**

### CODE REVIEW

File: docker/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+      #Atom Hopper version
+      version="1.9.1" \
+      description="Docker image for Cloudfeeds-Atom Hopper"
+
+#Saxon License file
+ARG saxon_lic
+ENV SAXON_HOME=/etc/saxon \
+
+    #The database type
+ENV DB_TYPE=H2 \
```
Comment (deep8103) [inline docker/Dockerfile:16]: This DB configuration is not used as of now as start.sh file is not used. We need to have a proper configuration for Postgres.

**Review by deep8103 — CHANGES_REQUESTED** [review]

**Comment by deep8103** [comment]
This PR should have been raised against a feature branch to safeguard us from blocking any production change that goes via the master.

File: docker/Dockerfile (hunk 1 above)
Comment (shub6691) [reply docker/Dockerfile:16]: done

File: docker/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    CATALINA_HOME=/opt/tomcat \
+    AH_HOME=/opt/atomhopper  \
+    PATH=${PATH}:${CATALINA_HOME}/bin:${AH_HOME}
+
+RUN mkdir -p "${CATALINA_HOME}" "${AH_HOME}" "${SAXON_HOME}" /etc/atomhopper/ /var/log/atomhopper/ /etc/cloudfeeds/translation/
+
+WORKDIR ${AH_HOME}
+
+COPY --from=tomcat /usr/local/tomcat ${CATALINA_HOME}
+COPY start.sh .
```
[the remark this answers is not in the document]
Comment (shub6691) [reply docker/Dockerfile:33]: removed

File: docker/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+COPY files/* /etc/atomhopper/
+COPY feedscatalog.xml /etc/feedscatalog/
+COPY ${saxon_lic} ${SAXON_HOME}/saxon-license.lic
+RUN chmod +x ${AH_HOME}/start.sh
+
+RUN apk --no-cache add curl tar bash \
+    && curl https://maven.research.rackspacecloud.com/content/repositories/releases/com/rackspace/usage/usage-schema/${APP_VERSION}/usage-schema-${APP_VERSION}-schema.tar.gz | tar xz \
+    && mv usage-schema-${APP_VERSION}/xslt-artifacts/* /etc/cloudfeeds/translation/. \
+    && curl -o atomhopper.war https://maven.research.rackspacecloud.com/content/repositories/releases/com/rackspace/feeds/feeds-atomhopper/${AH_VERSION}/feeds-atomhopper-${AH_VERSION}.war \
```
[the remark this answers is not in the document]
Comment (shub6691) [reply docker/Dockerfile:42]: done

File: docker/files/application-context.xml [hunk 4]
```diff
@@ -0,0 +1,4391 @@
...
+      <property name="prefixColumnMap">
+	<map>
+	  <entry key="type" value="eventtype"/>
+	  <entry key="tid"  value="tenantid"/>
+        </map>
+      </property>
+      <property name="delimiter" value=":"/>
+  </bean>
+
+<!--
```
[the remark this answers is not in the document]
Comment (shub6691) [reply docker/files/application-context.xml:94]: done.

File: docker/files/atom-server.cfg.xml [hunk 5]
```diff
@@ -0,0 +1,828 @@
...
+        <feed title="support feed" resource="/events">
+            <publisher reference="support-jdbc-new-feed-publisher" />
+            <feed-source reference="support-jdbc-new-feed-source" />
+            <feed-response-handlers>
+                <feed-response-handler reference="feed-paging-processor"/>
+                <feed-response-handler reference="role-aware-feed-etag-processor"/>
+            </feed-response-handlers>
+        </feed>
+    </workspace>
+    <!--
```
[the remark this answers is not in the document]
Comment (shub6691) [reply docker/files/atom-server.cfg.xml:35]: done

File: docker/repose-2018-saxon-license.lic [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+Issued=2018-01-12
+Series=S
+Serial=S006561
+User=P0001
+Evaluation=no
+Expiration=never
+UpgradeDays=335
+MaintenanceDays=335
+ 
+Signature=302C02145E39F15E0FD3C2FA6903AA4151D6BE836B80F28602146719D9C5EDFF90DD72018FB1A98A63044B3757DC
```
[the remark this answers is not in the document]
Comment (shub6691) [reply docker/repose-2018-saxon-license.lic:18]: done

File: pom.xml [hunk 7]
```diff
@@ -70,6 +70,12 @@
     <dependencyManagement>
         <dependencies>
 
+            <dependency>
+                <groupId>xerces</groupId>
+                <artifactId>xercesImpl</artifactId>
+                <version>2.11.0</version>
```
[the remark this answers is not in the document]
Comment (shub6691) [reply pom.xml:76]: done

File: docker/repose-2018-saxon-license.lic (hunk 6 above)
[the remark this answers is not in the document]
Comment (shub6691) [reply docker/repose-2018-saxon-license.lic:18]: done

**Pull request closed without merging**